case-lib/lib.sh

```diff
@@ -293,6 +293,33 @@ find_ldc_file()
     printf '%s' "$ldcFile"
 }
 
+func_logger_collect()
```

Comment:
```suggestion
func_sof_logger_collect()
```

We have too many logging tools, important to label them well.

Reply:
Updated.